### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BirdboyBolu**
Thank you for the time you are putting into AlaSQL!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ console.log(res); // [{a:2,b:6},{a:1,b:5},{a:3,b:4}]
 ```
 
 
-__If you are familiar with SQL it should come as no surprise that proper use of indexes on your tables is essential to get good performance.__
+__If you are familiar with SQL, it should be no surprise that proper use of indexes on your tables is essential for good performance.__
```
Comment (mathiasrw) [inline README.md:104]: thank you fir this improvement

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 AlaSQL - _( [à la](http://en.wiktionary.org/wiki/%C3%A0_la) [SQL](http://en.wikipedia.org/wiki/SQL) ) [ælæ ɛskju:ɛl]_ - is an open source SQL database for JavaScript with a strong focus on query speed and data source flexibility for both relational data and schemaless data.  It works in the web browser, Node.js, and mobile apps.
 
-This library is designed for:
+This library is perfect for:
```
Comment (mathiasrw) [inline README.md:32]: Please revert this change

**Review by mathiasrw — CHANGES_REQUESTED** [review]

**Review by mathiasrw — COMMENTED** [review]
Looks good - thanks!

**Comment by BirdboyBolu** [comment]
Hello @mathiasrw. I hope you are doing great?
I want you to know that I would love to join AlaSQL and handle its technical documentation. It would be great if we can talk more about this. I am willing to join the team, even if you don't have the budget to hire a full-time technical writer. 

My email is ayomide27victor@gmail.com. I look forward to hearing from you soon. Thank you

**Pull request merged**